velox/exec/tests/utils/CMakeLists.txt

```diff
@@ -50,4 +50,5 @@ target_link_libraries(
   velox_tpch_connector
   velox_presto_serializer
   velox_functions_prestosql
-  velox_aggregates)
+  velox_aggregates
```

Comment:
Please order the link libraries alphabetically (also fixing some that are out of order).